src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -428,36 +326,54 @@ def create_mds(mds, previous_result, devs_to_reuse)
         end
       end
 
-      # Creates bcaches in the given devicegraph
+      # Creates bcaches
       #
-      # @param bcaches         [Array<Planned::Bcache>] List of planned MD arrays to create
-      # @param previous_result [Proposal::CreatorResult] Starting point
-      # @param devs_to_reuse   [Array<Planned::Partition, Planned::StrayBlkDevice>] List of devices
+      # @param bcaches [Array<Planned::Bcache>] List of planned MD arrays to create
+      # @param devs_to_reuse [Array<Planned::Partition, Planned::StrayBlkDevice>] List of devices
       #   to reuse
-      # @return                [Proposal::CreatorResult] Result containing the specified MD RAIDs
-      def create_bcaches(bcaches, previous_result, devs_to_reuse)
-        bcaches.reduce(previous_result) do |result, bcache|
-          backing_devname = find_bcache_member(bcache.name, :backing, previous_result, devs_to_reuse)
-          caching_devname = find_bcache_member(bcache.name, :caching, previous_result, devs_to_reuse)
+      #
+      # @return [Proposal::CreatorResult] Result containing the specified MD RAIDs
+      def create_bcaches(bcaches, devs_to_reuse)
+        bcaches.reduce(creator_result) do |result, bcache|
+          backing_devname = find_bcache_member(bcache.name, :backing, creator_result, devs_to_reuse)
+          caching_devname = find_bcache_member(bcache.name, :caching, creator_result, devs_to_reuse)
           new_result = create_bcache(result.devicegraph, bcache, backing_devname, caching_devname)
           result.merge(new_result)
         end
       end
 
-      # Creates NFS filesystems in the given devicegraph
+      # Creates volume groups
       #
-      # @param nfs_filesystems [Array<Planned::Nfs>] List of planned NFS filesystems
-      # @param previous_result [Proposal::CreatorResult] Starting point
+      # @param vgs [Array<Planned::LvmVg>] List of planned volume groups to add
+      # @param devs_to_reuse [Array<Planned::Partition, Planned::StrayBlkDevice>] List of devices
+      #   to reuse as Physical Volumes
+      #
+      # @return [Proposal::CreatorResult] Result containing the specified volume groups
+      def set_up_lvm(vgs, devs_to_reuse)
+        # log separately to be more readable
+        log.info "BEGIN: set_up_lvm: vgs=#{vgs.inspect}"
+        log.info "BEGIN: set_up_lvm: previous_result=#{creator_result.inspect}"
+        log.info "BEGIN: set_up_lvm: devs_to_reuse=#{devs_to_reuse.inspect}"
```

Comment:
hmm, if you want to use some keywords for easier filtering I suggest something more distinguish then `BEGIN:` I think that `set_up_lvm:` is enough for quick grepping of logs.

Reply:
This code was not added as part of this PR, I have only moved it. But ok, I will remove that "BEGIN".